### PR TITLE
chore(dashmate): configure mainnet

### PR DESCRIPTION
**Pull request by shumkov**
## Issue being fixed or feature implemented

We need to update mainnet settings for v1 release

## What was done?

- Updated config for mainnet
- Enabled platform for mainnet
- Removed migrations

## How Has This Been Tested?

None

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have added "!" to the title and described breaking changes in the corresponding section if my code contains any
- [x] I have made corresponding changes to the documentation if needed

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/dashmate/configs/defaults/getMainnetConfigFactory.js [hunk 1]
```diff
@@ -36,7 +36,42 @@ export default function getMainnetConfigFactory(homeDir, getBaseConfig) {
...
+            ],
+          },
+          mempool: {
+            timeoutCheckTx: '3s',
+            txEnqueueTimeout: '30ms',
+            txSendRateLimit: 100,
+            txRecvRateLimit: 120,
+          },
+          genesis: {
+            chain_id: 'dash-mainnet-1',
```
Comment (QuantumExplorer) [inline packages/dashmate/configs/defaults/getMainnetConfigFactory.js:71]: I believe this is put into every tenderdash block in the header, could we use something smaller?

File: packages/dashmate/configs/getConfigFileMigrationsFactory.js [hunk 2]
```diff
@@ -733,6 +734,26 @@ export default function getConfigFileMigrationsFactory(homeDir, defaultConfigs)
           });
         return configFile;
       },
+      '1.0.0': (configFile) => {
+        Object.entries(configFile.configs)
+          .forEach(([name, options]) => {
+            if (name === 'base') {
+              options.platform.drive.tenderdash.mempool = base.get('platform.drive.tenderdash.mempool');
+              options.platform.drive.tenderdash.genesis = base.get('platform.drive.tenderdash.gensis');
```
Comment (QuantumExplorer) [inline packages/dashmate/configs/getConfigFileMigrationsFactory.js:742]: gensis or genesis?

File: packages/dashmate/src/listr/tasks/setup/setupRegularPresetTaskFactory.js [hunk 3]
```diff
@@ -83,8 +83,7 @@ export default function setupRegularPresetTaskFactory(
 
           ctx.config = defaultConfigs.get(ctx.preset);
 
-          // TODO: We need to change this and enable platform on mainnet
-          ctx.config.set('platform.enable', ctx.isHP && ctx.config.get('network') !== PRESET_MAINNET);
+          ctx.config.set('platform.enable', ctx.isHP);
```
Comment (QuantumExplorer) [inline packages/dashmate/src/listr/tasks/setup/setupRegularPresetTaskFactory.js:86]: PRESET_MAINNET is never used

**Review by QuantumExplorer — CHANGES_REQUESTED** [review]

File: packages/dashmate/configs/defaults/getMainnetConfigFactory.js (hunk 1 above)
Comment (QuantumExplorer) [reply packages/dashmate/configs/defaults/getMainnetConfigFactory.js:71]: dashm would do.

Comment (shumkov) [reply packages/dashmate/configs/defaults/getMainnetConfigFactory.js:71]: I believe verbosity is more important here

Comment (shumkov) [reply packages/dashmate/configs/defaults/getMainnetConfigFactory.js:71]: @lklimek what will you say?

Comment (shumkov) [reply packages/dashmate/configs/defaults/getMainnetConfigFactory.js:71]: `dash-1`?

Comment (lklimek) [reply packages/dashmate/configs/defaults/getMainnetConfigFactory.js:71]: we can just do `dash-1`

**Review by lklimek — APPROVED** [review]

**Review by QuantumExplorer — APPROVED** [review]

**Pull request merged**